loleaflet/src/layer/tile/WriterTileLayer.js

```diff
@@ -308,11 +308,18 @@ L.WriterTileLayer = L.CanvasTileLayer.extend({
 	},
 
 	_addHighlightSelectedWizardComment: function(annotation) {
+		if (this.highlightadded === true) {
+			var map = this._map;
+			$('.ui-header.level-0.mobile-wizard').each(function() {
+				map._docLayer._removeHighlightSelectedWizardComment(this.annotation);
+			});
+		}
 		var annotations = this._map._docLayer._annotations;
 		var annotationList = annotations._items;
 		var lastChild = annotations.getLastChildIndexOf(annotation._data.id);
 
 		while (true) {
+			this.highlightadded = true;
```

Comment:
once you set it to true you are not setting it false anywhere which defeated the purpose of this variable. Also consider renaming this variable to `_highlightAdded`